rust/libraries/acars_connection_manager/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "acars_connection_manager"
-version = "1.0.10"
+version = "0.1.2"
```

Comment:
As a hack to get the output JSON version number to show the binary version I bumped all of the `libraries/*` cargo toml version to match the `bin/acars_router` version. Thoughts on this? I suppose it doesn't much matter, as I should actually fix the problem instead of hacking around it.

Reply:
The proper fix will be released in Cargo 1.64.0 (https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance, more specifically https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-workspacepackage-table) which should render the need to do that moot.

1.64.0 is due to be released 2022-09-22, so not long to wait there (see https://github.com/rust-lang/cargo/blob/master/CHANGELOG.md#cargo-164-2022-09-22)